.github/workflows/pr-checks.yml

```diff
@@ -50,6 +44,8 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: recursive
+      - name: Run flagd-testbed
+        run: docker run -d -p 8013:8013 -v ${{ github.workspace }}/test-harness/testing-flags.json:/testing-flags.json ghcr.io/open-feature/flagd-testbed:v0.2.1
```

Comment:
I'm curious why you did this with a run, and not a service.

Reply:
Because the checkout step is needed to pull `testing-flags.json` before we can mount it

Reply:
`testing-flags.json` is included in the docker image, and included in the entry point: https://github.com/open-feature/test-harness/blob/main/Dockerfile

Reply:
To test that changing the value in the config invalidates the cache the host machine needed to alter that file. I figured the simplest way was by mounting the host’s config file and then any alterations done to the host file are reflected in the container

Reply:
Ah, right. That makes sense. I forgot about the fact that the config needs to be mutated for these tests. Of course.